includes/class-reader-activation.php

```diff
@@ -1184,5 +1189,37 @@ function( $type ) {
 
 		return $emails;
 	}
+
+	/**
+	 * Get the from email address used to send all transactional emails.
+	 * We avoid use of the `wp_mail_from` hook because we only want to
+	 * set the email address for Reader Activation emails, not all emails
+	 * sent via wp_mail.
+	 */
+	public static function get_from_email() {
+		// Get the site domain and get rid of www.
+		$sitename   = wp_parse_url( network_home_url(), PHP_URL_HOST );
+		$from_email = 'no-reply@';
```

Comment:
I'm also fine with `notifications@site.com`, as suggested by @yogeshbeniwal

Reply:
I like `no-reply` for now since it signals that no one will be monitoring that inbox for replies. The value is filterable via the `newspack_reader_activation_from_email` hook if needed.

Reply:
Thank you. Will override it at site wide as done for other emails.